src/components/app/ErrorBoundary.js

```diff
@@ -61,35 +68,26 @@ export class ErrorBoundary extends React.Component<Props, State> {
     });
   }
 
-  _toggleErrorDetails = () => {
-    this.setState((state) => ({ showDetails: !state.showDetails }));
-  };
-
   render() {
     if (this.state.hasError) {
-      const { errorString, componentStack, showDetails } = this.state;
+      const { errorString } = this.state;
       return (
         <div className="appErrorBoundary">
           <div className="appErrorBoundaryContents">
             <div className="photon-message-bar photon-message-bar-error photon-message-bar-inner-content appErrorBoundaryMessage">
               <div className="photon-message-bar-inner-text">
                 {this.props.message}
+                {errorString ? ` (${errorString})` : null}.{' '}
+                {this.props.reportExplanationMessage}
```

Comment:
@flodolo 
I know that the localization isn't perfect, because we concatenate strings instead of providing them in their entirety. The reason is that I didn't want to spend too much time on it, and I thought that because this is an error component that the users shouldn't see much that would be good enough. I hope that's good enough for you!

Reply:
Can you help me understand where the `Error: ` part come from?

I'm really not a fan of having it added as part of the first sentence. Can we keep using `Details--error-boundary-message`, and insert (concatenate) a fully-formed separate sentence `Error: foo. ` instead? Or maybe put that at the very end, between the localized paragraph and the button?

Reply:
This comes from calling `toString` on the error object. I this case the error object was a `new Error("foo")` but usually it's an Error with more context (hopefully).

Reply:
Thanks for the feedback, I think this is indeed better now:

![image](https://user-images.githubusercontent.com/454175/225334394-3fa853e3-ed82-4772-91a9-2c5845ed2029.png)

Telll me what you think :-)